src/tlo/methods/healthburden.py

```diff
@@ -116,10 +119,21 @@ def initialise_simulation(self, sim):
         self.process_causes_of_disability()
         self.process_causes_of_dalys()
 
-        # 4) Launch the DALY Logger to run every month, starting with the end of the first month of simulation
+        # 4) Launch the DALY and Prevalence Logger to run every month, starting with the end of the first month of simulation
+        # 5) Schedule `Healthburden_WriteToLog` that will write to log annually
         sim.schedule_event(Get_Current_DALYS(self), sim.date + DateOffset(months=1))
+        if self.parameters['logging_frequency_prevalence'] == 'day':
+            sim.schedule_event(Get_Current_Prevalence(self), sim.date + DateOffset(days=0))
+            sim.schedule_event(Healthburden_WriteToLog_Prevalences(self), sim.date + DateOffset(days=0))
```

Comment:
if these two events happen at the same frequency and we want to guarantee the order they happen in, I think they should be ONE event.